src/main/java/org/jabref/logic/journals/JournalInformationFetcher.java

```diff
@@ -0,0 +1,229 @@
+package org.jabref.logic.journals;
+
+import java.util.ArrayList;
+import java.util.Comparator;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Optional;
+import java.util.Set;
+import java.util.regex.Pattern;
+
+import javafx.util.Pair;
+
+import org.jabref.logic.importer.FetcherException;
+import org.jabref.logic.importer.WebFetcher;
+import org.jabref.logic.l10n.Localization;
+import org.jabref.model.entry.identifier.ISSN;
+
+import kong.unirest.HttpResponse;
+import kong.unirest.JsonNode;
+import kong.unirest.Unirest;
+import kong.unirest.json.JSONArray;
+import kong.unirest.json.JSONException;
+import kong.unirest.json.JSONObject;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+/**
+ * Fetches journal information from the JabRef Web API
+ */
+public class JournalInformationFetcher implements WebFetcher {
+    public static final String NAME = "Journal Information";
+    private static final Logger LOGGER = LoggerFactory.getLogger(JournalInformationFetcher.class);
+    private static final String API_URL = "https://jabref.org/api";
+    private static final Pattern QUOTES_BRACKET_PATTERN = Pattern.compile("[\"\\[\\]]");
+
+    @Override
+    public String getName() {
+        return JournalInformationFetcher.NAME;
+    }
+
+    public Optional<JournalInformation> getJournalInformation(String issnString, String journalName) throws FetcherException {
+        ISSN issn = new ISSN(issnString);
+        String cleanedISSN = "";
+
+        if (issn.isValidFormat() || issn.isCanBeCleaned()) {
+            cleanedISSN = issn.getCleanedISSN();
+        } else {
+            LOGGER.warn(Localization.lang("Incorrect ISSN format"));
+        }
+
+        Optional<JournalInformation> journalInformationOptional = Optional.empty();
+
+        JSONObject postData = buildPostData(cleanedISSN, journalName);
```

Comment:
@tobiasdiez I did not see that during the first reviews. This is Level 0 in the [Richardson Maturity Model](https://martinfowler.com/articles/richardsonMaturityModel.html). Maybe for now, this is OK. Regarding the aim of education of properly designed APIs, we might consider an udpate on the http resource design. -- I think, this is a longer topic and does not need to be solved now. It is more an input for https://github.com/JabRef/jabref/pull/9748.

Reply:
Because all the api endpoints are graphql endpoints and there is no point in making an exception for the journal infos. Now that jabref no longer has the library limits, we can work on using a proper java graphql client but for this PR I think its good enough to use http + manual parsing (which is actually not that different than using a rest call)